gulp/makeCss.js

```diff
@@ -62,3 +62,7 @@ gulp.task("copyCssDeps", function() {
     )
     .pipe(gulp.dest("./src/scss/cssIncludes"));
 });
+gulp.task('copyCssImages', function() {
+   gulp.src('./node_modules/leaflet/dist/images/*.png')
+   .pipe(gulp.dest(paths.bundleDir+'/images'));
+});
```

Comment:
it looks like you could revert the gulp makeCss changes, and update the scss file here instead:
https://github.com/OpenTriply/YASGUI.YASR/blob/gh-pages/src/leaflet.scss#L5 (with background-image snippet from above)